### ISSUE TITLE
Radio Button Focus is not displayed

### ISSUE THREAD

**Issue by mscherotter** [adobe/spectrum-css#1675]
## Description
Keyboard focus (blue circle) is not being displayed for radio buttons.
![image](https://user-images.githubusercontent.com/641702/225735404-10348fe6-a106-4bb5-92cf-3466b991baa9.png)

This page demonstrates the issue for non-spectrum radio buttons, you can see the keyboard focus but you cannot for Spectrum radio buttons.
https://pippasgallery97-slot1.azurewebsites.net/spectrumTest.html

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):**

**Comment by pfulton** [comment]
Radio has been migrated to use the `@adobe/spectrum-tokens` system, and it has also received a refactor to use `:focus-visible`. Additionally, we've checked the [Docs site](https://opensource.adobe.com/spectrum-css/radio.html) as well as our Storybook implementation and this is working as intended.

Please let us know if you have further issues with the focus indicator.
